doc/man1/openssl-req.pod.in

```diff
@@ -497,14 +497,15 @@ any digest that has been set.
 This option masks out the use of certain string types in certain
 fields. Most users will not need to change this option.
 
-It can be set to several values B<default> which is also the default
-option uses PrintableStrings, T61Strings and BMPStrings if the
-B<pkix> value is used then only PrintableStrings and BMPStrings will
-be used. This follows the PKIX recommendation in RFC2459. If the
-B<utf8only> option is used then only UTF8Strings will be used: this
-is the PKIX recommendation in RFC2459 after 2003. Finally the B<nombstr>
-option just uses PrintableStrings and T61Strings: certain software has
-problems with BMPStrings and UTF8Strings: in particular Netscape.
+It can be set to several values: B<utf8only> uses only UTF8Strings: this is
+the default and the PKIX recommendation in RFC2459 after 2003; the B<default>
+value (which is not the default!) uses BMPStrings, IA5Strings,
```

Comment:
This is still a bit of a confusing read in regards to defaults due to their being 2 defaults :) I think making it more clear would be a good idea..

You could even indent the options maybe?

````
This option masks out the use of certain string types in certain
fields. It can be set to one of the following:
=over 4

=item B<utf8only>
Uses only UTF8Strings. This is the PKIX recommendation in RFC2459 after 2003.

=item ....
.....

=back

This option masks out the use of certain string types in certain
fields.

If this option is not set it will use B<utf8only> by default. Most users will not need to change this option.
````

I dont see the harm in leaving the comment regarding Netscape in there also.

Reply:
It looks like it also handles MASK:numericmask inside
ASN1_STRING_set_default_mask_asc() but that may be better left undocumented.

Reply:
There are quite a few B_ASN1_ flag values.. How are you figuring out what the subset is... (Since the mask sets all of them)

Reply:
I took another stab at wording this, can you take a look?